method/src/main/java/net/kyori/event/method/annotation/Subscribe.java

```diff
@@ -41,9 +41,10 @@
 @Target(ElementType.METHOD)
 public @interface Subscribe {
   /**
-   * Gets the post order.
+   * Gets the post order of this subscriber.
    *
    * @return the post order
+   * @see PostOrder
    */
-  PostOrder value() default PostOrder.NORMAL;
+  int postOrder() default PostOrder.FIRST;
```

Comment:
Two things:
- why `FIRST` now?
- I'd prefer to keep the name `value` so it can be written as: `@Subscribe(PostOrder.LAST)`

Reply:
First was a mistake.

Second: it was changed because previously, the type `PostOrder` made it clear that `value()` was defining the post order. However, that isn't the case anymore. `int value()` could mean anything.